content/thesis-topics.md

```diff
@@ -29,3 +29,12 @@ You will usually be supervised by the PhD students and graded + supervised by Pr
 - Analysis of the attack surface of infotainment systems via the broadcast interface [[2]](https://arstechnica.com/cars/2022/02/radio-station-snafu-in-seattle-bricks-some-mazda-infotainment-systems/)
 - Analysis of log files from automotive control units for forensic use
 - Fuzzing of Bluetooth implementations (A2DP, PBAP, SDAP, ...) 
+
+### Digital Forensics Topics 
+
+| Title | Short description | Contact person |
+| ----- | ----------------- | -------------- | 
+| File carving using Large Language Models (LLMs) | File carving of different memory dumps and using LLMs to identify traces | [kevin.mayer@carissma.eu](kevin.mayer@carissma.eu) |
```

Comment:
Are you sure these links work? Shouldn't they be `mailto:` links?

Reply:
Consider using the included `cloakemail` plugin: `{{<cloakemail display="display text" address="email address">}}`

Reply:
done